src/targets/services/attributesHelpers.spec.js

```diff
@@ -0,0 +1,73 @@
+import {
+  findNewAttributes,
+  updateMyselfWithIdentity
+} from './attributesHelpers'
+
+describe('attributesHelpers', () => {
+  const newIdentity = {
+    contact: { name: 'John', age: 32 }
+  }
+  let currentMyselfContactMock
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+    currentMyselfContactMock = {
+      id: 1
+    }
+  })
+
+  const contactCollection = {}
+  contactCollection.update = jest.fn()
+  const mockLog = jest.spyOn(console, 'log')
+
+  describe('when there are new attributes', () => {
+    it('should be found', () => {
+      const newAttributesFound = findNewAttributes(
+        newIdentity.contact,
+        currentMyselfContactMock
+      )
+      expect(Object.keys(newAttributesFound).length).toBe(2)
+      expect(newAttributesFound).toEqual({ name: 'John', age: 32 })
+    })
+  })
+
+  describe('when there are attributes', () => {
+    it('if there are new ones, they should be added in currentMySelfContact', async () => {
+      expect(currentMyselfContactMock).toEqual({ id: 1 })
```

Comment:
Is this needed ?

Reply:
Yes I think, if there is an other value it can beak the test